.github/workflows/check-kotlin-gradle.yml

```
@@ -47,7 +52,7 @@ on:
 jobs:
   check-and-report:
     name: Lint, Test And Reports
-    runs-on: [self-hosted, Linux, X64, mortar-runner]
+    runs-on: ${{ (inputs.mortar-version == 'v1') && fromJSON('[ "self-hosted", "Linux", "X64", "mortar-runner" ]') || fromJSON('[ "self-hosted", "Linux", "X64", "mortar", "v2" ]') }}
```

```suggestion
    runs-on: [ "self-hosted", "Linux", "X64", "mortar", ${{ inputs.mortar-version }}]
```
v1도 label 처리를 해 둬서 요렇게만 해도 괜찮을 것 같습니다.

---

이것도 되나보군요. 바꿔서 해보겠습니다!